pkg/build/builder/sti.go

```diff
@@ -265,8 +267,8 @@ func (s *S2IBuilder) Build() error {
 		} else {
 			glog.V(2).Infof("No push secret provided")
 		}
-		glog.V(1).Infof("Pushing %s image ...", pushTag)
-		if err := pushImage(s.dockerClient, pushTag, pushAuthConfig); err != nil {
+		fmt.Fprintf(s.out, "Pushing %s image ...\n", pushTag)
```

Comment:
so w/ existing master, if i do openshift start w/ no loglevel, i get this message:
```
Creating a new S2I builder with build config:  blah blah blah.....
```
which is logged at level 2:
```
	glog.V(2).Infof("Creating a new S2I builder with build config: %#v\n", describe.DescribeConfig(config))
```